### PR TITLE
Zoey_Completed javascript-primitives-and-variables

### PR DESCRIPTION
**Pull request by Zoey1318**
javascript-primitives-and-variables practice

### CODE REVIEW

File: javascript-primitives-and-variables/main.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+console.log('typeof totalPets:', typeof totalPets);
+
+var unknown = null;
+console.log('value of unknown:', unknown);
+console.log('typeof unknown:', typeof unknown);
+
+var never;
+console.log('value of never:', never);
+console.log('typeof never:', never);
```
Comment (scrumptiousdev) [inline javascript-primitives-and-variables/main.js:22]: For this one, we need to check type of never

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
One small change and you are good!

**Comment by Zoey1318** [comment]
re javascript-primitives-and-variables practice

**Review by scrumptiousdev — APPROVED** [review]
🪄 Approved!

**Pull request merged**